package.json

```diff
@@ -87,7 +92,9 @@
     "@aws-cdk/aws-logs": "1.x",
     "@aws-cdk/aws-sns": "1.x",
     "@aws-cdk/aws-sns-subscriptions": "1.x",
-    "@aws-cdk/core": "1.x"
+    "@aws-cdk/core": "1.x",
+    "esbuild": "0.x",
+    "node-fetch": "2.6.x"
```

Comment:
Should this also be something like `>= 2.6.1`. Now if node-fetch comes with 2.7 version it is not supported by mca-monitoring

Reply:
we should use caret here as well, `>=` would also allow 3.x.x